Userland/Applications/Calendar/CalendarWidget.cpp

```diff
@@ -93,7 +93,7 @@ ErrorOr<NonnullRefPtr<CalendarWidget>> CalendarWidget::create(GUI::Window* paren
 
     file_menu->add_separator();
 
-    file_menu->add_action(GUI::CommonActions::make_quit_action([&](auto&) {
+    file_menu->add_action(GUI::CommonActions::make_quit_action([widget](auto&) {
```

Comment:
What was the problem here?

Reply:
Capturing the `widget` local by reference rather than by value (see commit message)

Reply:
Oh sorry, for one-commit PRs github's pull request UI usually copies the commit text body into the PR description. Since that was empty, I blindly assumed there's no commit message other than the first line. Apologies for not checking, and thanks for explaining!